packages/api/endpoints/pdrs.js

```diff
@@ -39,10 +40,12 @@ async function list(req, res) {
 async function get(req, res) {
   const pdrName = req.params.pdrName;
 
-  const pdrModel = new models.Pdr();
+  const knex = await getKnexClient({ env: process.env });
```

Comment:
I think it defaults `env` to `process.env`, so I'm not sure this is necessary?

Reply:
https://github.com/nasa/cumulus/pull/2273/commits/7ed58e6a1895c91c2d21825e119ebfa9ace452bc